delivery_package_default_shipping_weight/models/product_packaging.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    package_default_shipping_weight = fields.Float(
```

Comment:
IMHO, we should use digits here as :

https://github.com/odoo/odoo/blob/14.0/addons/product/models/product.py#L119

Reply:
The code is updated